android/assets/jsons/translations/Czech.properties

```diff
@@ -29,13 +29,13 @@ Fastlane_full_description =
 
 # General
  # Requires translation!
-Delete = 
+Delete = Odstranit
 
 # Base ruleset names
  # Requires translation!
-Civ V - Vanilla = 
+Civ V - Vanilla = Civ V - Vanilla
  # Requires translation!
-Civ V - Gods & Kings = 
+Civ V - Gods & Kings = Civ V - Vanilla
```

Comment:
Civ V - Gods & Kings shouldn't be translated with Civ V - Vanilla

Reply:
Oops, messed that up. Fixing it now